.editorconfig

```diff
@@ -0,0 +1,261 @@
+root = true
+
+[*]
```

Comment:
Lacks pretty important settings, like `max_line_length`, `end_of_line` type and so on... :(

Reply:
Added max_line_length, end_of_line and indent_style to editor config